cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -1723,7 +1723,7 @@ def test_image_import(self):
         course = courses[0]
 
         # Make sure the course image is set to the right place
-        self.assertEqual(course.course_image, 'images_course_image.jpg')
+        self.assertEqual(course.course_image, '')
```

Comment:
new default is empty string, because there is the default image that shows up if none is set.